draft-ietf-quic-recovery.md

```diff
@@ -522,9 +524,14 @@ removed from bytes in flight when the Initial and Handshake keys are discarded.
 ### Sending Probe Packets
 
 When a PTO timer expires, a sender MUST send at least one ack-eliciting packet
-as a probe, unless there is no data available to send.  An endpoint MAY send up
-to two full-sized datagrams containing ack-eliciting packets, to avoid an
-expensive consecutive PTO expiration due to a single lost datagram.
+in the packet number space as a probe, unless there is no data available to
+send.  An endpoint MAY send up to two full-sized datagrams containing
+ack-eliciting packets, to avoid an expensive consecutive PTO expiration due
+to a single lost datagram.
```

Comment:
Confirmation question: does this mean that we'd be permitting more datagrams to be sent in effectively one PTO, when multiple packet number spaces are involved?

Consider the case where the PTO is armed for both the Handshake packet number space and the 1-RTT packet number space. Now that we have a PTO timer for each of the packet number space, we'd see both of them firing, and the text reads that an endpoint is allowed to send two datagrams for each.

Reply:
Good point, I updated the PR to try to limit the number of packets sent when implementations did not coalesce.